docs/styles/spec/drawer-spec.less

```diff
@@ -0,0 +1,12 @@
+.drawer-spec {
+  .dialog-helpers {
```

Comment:
Positions disclosures for page modal and page popovers so that there's a good chance that at least one popover will overlap the drawer.